### ISSUE TITLE
Wrong content in /topics command

### ISSUE THREAD

**Issue by nxblnd** [dnlmurray/morningpaper#1]
It is possible to write garbage as topic name. It is not inserted into database, but could be confusing for users.

**Comment by dnlmurray** [comment]
It depends on what you mean by "topics". API has two types of clarifying parameters: category and keywords. If you mean categries, then making users choose from limited list of topics would be a better option, since NewsAPI understands very few of them: _business_, _entertainment_, _general_, _health_, _science_, _sports_, and _technology_.

If you mean keywords, on the other hand, I wouldn't say that it's a good idea, considering our approach. Keywords will make every API output extremely personified and using db in this case seems pointless to me.

**Comment by nxblnd** [comment]
Ok, we should really have synchronized terms in DB with APIs. "Topics" mean categories. 

As things work right now, on /topics command special Telegram keyboard is shown. But all it does is sending specified messages. No one is blocking user from sending something they type on their own. So, it is possible that bot answers with

> _You have chosen the following topics: ывапывп, business_

As i said, this does not get into DB, so it is not critical, but can lead to worse UX.

**Comment by dnlmurray** [comment]
Can't you make a check if user input matches any existing category? We could just drop any non-valid categories in set_topics(). That'd solve the problem in my opinion.

**Comment by nxblnd** [comment]
Yeah, this is solution. But i've wanted to do some other things, so i created this issue as a reminder
